### ISSUE TITLE
Retrieve full calendar history

### ISSUE THREAD

**Issue by mcwieger** [Ericsson/exchangecalendar#238]
By default not all events are retrieved from the Exchange server (only those that are within a number of months before and after current date). When going back or forward in the calendar (e.g. selecting 1 January 2014), all events between Today and that date are retrieved (text '1 running and x queued jobs' appears in the bottom right of the screen).

Is it possible to retrieve all events, or all events between two given dates, by default (so without having to manually navigate)?

Reason for the request is that I export and publish the calendar to a shared CalDav server, which now displays incomplete information.

**Comment by bavincen** [comment]
Yes You can go to cache settings and increase the cache period. it is not very good idea to retrieve all events at one go. may hang tb

**Comment by mcwieger** [comment]
That does not appear to solve the issue: when I export the calendar, it still only exports one month ahead. Any other ideas?

**Comment by bavincen** [comment]
in progress..
